docs/content/preview/explore/ysql-language-features/pg-extensions.md

```diff
@@ -638,6 +720,8 @@ SELECT hll_cardinality(set) FROM helloworld WHERE id = 1;
 (1 row)
 ```
 
+For a more advanced example, see the [Data Warehouse Use Case](https://github.com/yugabyte/yugabyte-db/tree/master/src/postgres/third-party-extensions/postgresql-hll#data-warehouse-use-case).
+
```

Comment:
pgsql-postal extension is also missing the intro: https://deploy-preview-17664--infallible-bardeen-164bc9.netlify.app/preview/explore/ysql-language-features/pg-extensions/#pgsql-postal-example